openhands-tools/openhands/tools/delegate/definition.py

```diff
@@ -0,0 +1,143 @@
+"""Delegate tool definitions for OpenHands agents."""
+
+from collections.abc import Sequence
+from typing import TYPE_CHECKING, Literal
+
+from pydantic import Field
+from rich.text import Text
+
+from openhands.sdk.llm.message import ImageContent, TextContent
+from openhands.sdk.tool.tool import (
+    Action,
+    Observation,
+    ToolAnnotations,
+    ToolDefinition,
+)
+
+
+if TYPE_CHECKING:
+    pass
+
+
+class DelegateAction(Action):
+    """Action for delegating tasks to sub-agents."""
+
+    operation: Literal["spawn", "send", "close"] = Field(
+        description="The delegation operation to perform"
+    )
+    task: str | None = Field(
+        default=None, description="Task description for spawn operation"
+    )
+    sub_conversation_id: str | None = Field(
+        default=None,
+        description="ID of the sub-conversation for send/close operations",
+    )
+    message: str | None = Field(
+        default=None, description="Message to send to sub-agent (for send operation)"
+    )
```

Comment:
I wonder if we can just keep `message` and remove `task` -- since a `task` can just be a message